### ISSUE TITLE
[SFSF]Carousel: A11y - ARIA role should be appropriate for the element

### ISSUE THREAD

**Issue by VidhiNPatel** [SAP/ui5-webcomponents-react#7277]
### Describe the bug

ARIA role should be appropriate for the element

### Reproduction steps

1. Download the axe DevTools - Web Accessibility Testing Chrome extension
2. Go to https://sap.github.io/ui5-webcomponents-react/v2/?path=/docs/layouts-floorplans-carousel--docs 
3. Open the web inspect -> Select Axe DevTool -> Full Page Scan
4. You will see ARIA role should be appropriate for the element issue reported by Axe tool

### Expected Behaviour

https://dequeuniversity.com/rules/axe/4.10/aria-allowed-role?application=AxeChrome

### Screenshots or Videos

<img width="1695" alt="Image" src="https://github.com/user-attachments/assets/ab10207a-5428-42b6-ba47-7ccc48e614fc" />

### UI5 Web Components for React Version

2.9.2

### UI5 Web Components Version

2.9.0

### Browser

Chrome

### Relevant log output

```shell

```

### Declaration

- [x] I’m not disclosing any internal or sensitive information.

**Comment by Lukas742** [comment]
Thanks for reporting! I'll forward this issue to our UI5 Web Components Colleagues as the affected component is developed in their repository.
